core/src/main/java/org/infinispan/persistence/spi/MarshalledValue.java

```diff
@@ -10,11 +12,16 @@
  * @since 10.0
  */
 public interface MarshalledValue {
+
+   @ProtoField(number = 1, name = "value", javaType = ByteBufferImpl.class)
```

Comment:
Yes. Originally I was thinking that by adding it to the SPI it would mean that the proto format would be applicable to custom implementations of the interface, but we have to register the concrete class with the PersistenceContextInitializer anyway, so this doesn't work.